apps/explorer/src/ui/ImageIcon.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva, type VariantProps } from 'class-variance-authority';
+import { useState } from 'react';
+
+const imageStyle = cva(
+    [
+        'bg-gradient-to-r from-gradient-blue-start to-gradient-blue-end text-white capitalize overflow-hidden',
+    ],
+    {
+        variants: {
+            size: {
+                sm: 'w-6 h-6 font-medium text-subtitleSmallExtra',
+                md: 'w-7.5 h-7.5 font-medium text-body',
+                lg: 'md:w-10 md:h-10 w-8 h-8 font-medium text-heading4 md:text-3xl',
```

Comment:
Can we avoid using `text-3xl`? It's a tailwind default instead of from our design system.